tests/tasks/assert_corosync_config.yml

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Fetch corosync configuration from the cluster
+  # Remove the contents of 'Nodes:' section as it depends on
+  # testing environment
+  shell:
+    cmd: pcs cluster config | sed -e '/^Nodes:$/,/^[^ ]/ { /^ /d }'
+  register: __test_pcs_corosync_config
+  changed_when: no
+
+- name: Print real corosync configuration
+  debug:
+    var: __test_pcs_corosync_config.stdout_lines
```

Comment:
if you want users to be able to hide this, I suggest using `verbosity`: https://docs.ansible.com/ansible/2.9/modules/debug_module.html#debug-module
e.g. using `verbosity: 2` means that the user would need to run ansible with `-vv` in order to see this output

however, if you think that this should always be in the output, that's fine too - we have had users complain about the amount of output in some of our roles - the storage role team recently merged a change to reduce the amount of output by default - https://github.com/linux-system-roles/storage/pull/256

Reply:
nevermind - this is in test code, so yes, you always want to see this